tests/standalone/Boto3Using.py

```diff
@@ -0,0 +1,103 @@
+import boto3
+from moto import mock_s3
+
+# nuitka-skip-unless-imports: boto3,moto
+
+def getClient():
+    """
+    returns the boto3 client
+    """
+    return boto3.client('s3')
+
+def listS3Buckets():
+    """
+    lists s3 bucket names
+    """
+    s3 = getClient()
+
+    response = s3.list_buckets()
+    if response:
+        for bucket in response.get('Buckets', []):
+            yield bucket['Name']
+
+def listS3Objects(bucket):
```

Comment:
Method names should be written in `snake_case`.
This comment applies to other methods that you have added as well.
Example: `listS3Objects ` -> `list_s3_objects`, `readS3Object ` -> `read_s3_object`

Reply:
In my pull request #314 @kayhayen said to use camelCase for methods, which one should I use?